packages/tests/integration/src/EntityManager.txns.test.ts

```diff
@@ -1,171 +0,0 @@
-        await txn.commit();
-      });
-    })();
-
-    const t2 = (async () => {
-      const txn = await knex.transaction();
-      try {
-        await txn.raw("set transaction isolation level serializable;");
-        await steps.on(2, () => txn.select("*").from("publishers").where({ name: "foo" }));
-        await steps.on(4, () =>
-          txn.insert({ name: "foo", base_sync_default: "foo", base_async_default: "foo" }).into("publishers"),
-        );
-      } catch (e) {
-        await txn.rollback();
-        throw e;
-      }
-    })();
-
-    await t1;
-    await expect(t2).rejects.toThrow("could not serialize access");
-
-    const rows = await knex.select("*").from("publishers");
-    expect(rows.length).toEqual(1);
-    expect(rows).toMatchObject([{ id: 1, name: "foo" }]);
-  });
-
-  it("using serializable via node-pg avoids anomalies", async () => {
-    const steps = new Stepper();
-
-    const pool = new Pool(newPgConnectionConfig());
-
-    const t1 = (async () => {
-      const conn = await pool.connect();
-      await conn.query("begin");
-      await conn.query("set transaction isolation level serializable;");
-      await steps.on(1, async () => conn.query("select * from publishers where name = 'foo'"));
-      await steps.on(3, async () => {
-        await conn.query(
-          "insert into publishers (name, base_sync_default, base_async_default) values ('foo', 'foo', 'foo')",
-        );
-        await conn.query("commit");
-      });
-      conn.release();
-    })();
-
-    const t2 = (async () => {
-      const conn = await pool.connect();
-      try {
-        await conn.query("begin");
-        await conn.query("set transaction isolation level serializable;");
-        await steps.on(2, async () => conn.query("select * from publishers where name = 'foo'"));
-        await steps.on(4, async () =>
-          conn.query(
-            "insert into publishers (name, base_sync_default, base_async_default) values ('foo', 'foo', 'foo')",
-          ),
-        );
-      } finally {
-        conn.release();
-      }
-    })();
-
-    await t1;
-    await expect(t2).rejects.toThrow("could not serialize access");
-    await pool.end();
-
-    const rows = await select("publishers");
-    expect(rows.length).toEqual(1);
-    expect(rows).toMatchObject([{ id: 1, name: "foo" }]);
-  });
-
-  it("using EntityManager.transaction avoids anomalies", async () => {
-    const steps = new Stepper();
-
-    const t1 = (async () => {
-      const em = newEntityManager();
-      await em.transaction(async () => {
-        await steps.on(1, async () => em.find(Publisher, { name: "foo" }));
-        await steps.on(3, async () => em.create(SmallPublisher, { name: "foo", city: "c1" }));
-      });
-      await steps.on(5, async () => {});
-    })();
-
-    const t2 = (async () => {
-      const em = newEntityManager();
-      await em.transaction(async () => {
-        await steps.on(2, async () => em.find(Publisher, { name: "foo" }));
```

Comment:
I'm kinda wanting to keep this test file, just to play with/repro anomaly failures in the future, even if we do totally remove the isolationLevel capability from the EM.

Also, I think technically this test shows how users _could_ have used the isolation level feature, by explicitly doing their em.finds/loads within the txn as well.

Granted, we don't do that at all, but apps I used in the past did (it was common before "Internet scale" to always do reads and writes in a txn).

Reply:
(afk dropped some of my comment...)

Maybe we keep this test, but update it to fail, to show we explicitly don't support it atm?